pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -1331,3 +1351,27 @@ func TestDeploymentConfigChanged(t *testing.T) {
 		}
 	}
 }
+
+func TestDurationToHAProxyTimespec(t *testing.T) {
+	testCases := []struct {
+		inputDuration  time.Duration
+		expectedOutput string
+	}{
+		{100 * time.Millisecond, "100ms"},
```

Comment:
Can we add the expectations for zero.

Reply:
Actually, does that (a zero value) make any sense? Just looking for coverage here.

Reply:
It shouldn't get called for a duration of 0, at least at the moment, but for completeness it probably should be covered.

Reply:
Yeah, I consider this a utility function. I don't know what or who the callers are, but "full coverage" helps its cause. Thanks.

Reply:
Done